taxi/urls.py

```diff
@@ -1,6 +1,8 @@
+from argparse import Namespace
```

Comment:
Remove all changes from this file
There no need Namespace, auth_views, defining path for logout or login. Just revert all changes in this file